assets/css/style.css

```diff
@@ -1,4 +1,29 @@
 html, body{
     padding: 0;
     margin: 0;
+}
+
+.title {
+    font-family: 'Pacifico', cursive;
+}
+
+.is-divider {
+    font-family: 'Montserrat', sans-serif;
+}
+
+.city-input {
+    height: 350px;
+}
+
+.box {
```

Comment:
I was leaving the styling off the the map area because I didn't know the size, but if it works like this that's great.